api/serializers.py

```diff
@@ -347,6 +348,13 @@ def validate(self, attrs):
                     _("Cannot add a vacation or sick shift to a workday with other shifts.")
                 )
 
+        # validate feirtage is just clackable on a feiertag
+        de_he_holidays = holidays.country_holidays('DE', subdiv='HE')
+        if started.strftime("%d/%m/%Y") in de_he_holidays and type is not 'bh':
```

Comment:
Das muss in die `validate_started` Methode.

In die `.validate()` methode kommen nur Validierungen in denen man mehr als Feld benötigt.
Beispiel: Validiere dass `stopped` nicht vor `started` stattfindet (Eine Scicht darf nicht Enden bevor sie begonnen hat.)

In diesem Falls wird nur `started` benötigt.